packages/amplify-java-template-provider/resources/lambda/hello-world/HelloPojo.java.ejs

```diff
@@ -0,0 +1,14 @@
+<%= props.topLevelComment %>
+
+package example;
+
+import com.amazonaws.services.lambda.runtime.Context; 
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+
+public class HelloPojo implements RequestHandler<RequestClass, ResponseClass>{   
```

Comment:
This isn't a POJO

Reply:
I know this is not POJO , but the underlying request and response classes , this is just an example to show POJO example